src/toxcore/messenger/mod.rs

```diff
@@ -1,4 +1,6 @@
 /*! The implementation of Messenger
 */
 
-pub mod packet;
+mod packet;
+
+pub use self::packet::*;
```

Comment:
I prefer `pub mod packet;` over `mod packet;` + `pub use self::packet::*`;

It's our common style to do so. CHeck out https://github.com/tox-rs/tox/blob/master/src/toxcore/tcp/mod.rs https://github.com/tox-rs/tox/blob/master/src/toxcore/dht/mod.rs#L4 and so on

Reply:
done